opm/output/data/Cells.hpp

```diff
@@ -52,6 +52,10 @@ namespace data {
        will not be output anywhere else.
 
      INIT : Fields which should go to the INIT file.
+
+     RESTART_OPM_EXTENDED: Cell-based quantities that are specific to
+       OPM-Flow.  Output only to extended OPM restart files.  Specifically
+       not output to ECLIPSE-compatible restart files.
```

Comment:
I think the `RESTART_AUX` are raw double values - do you envisage the same for the `RESTART_OPM_EXTENDED`?

Reply:
> I think the `RESTART_AUX` are raw double values

That may have been the original intention, but it is not how the system has behaved for a long time.  First the [unit conversion](https://github.com/OPM/opm-common/blob/1ac240a425c64c3b045d28fd963141e04d333f90/src/opm/output/data/Solution.cpp#L51-L73) support of `data::Solution` does not take these tags into account, meaning all vectors are subject to unit conversion.  Second, even in current master, the auxiliary vectors honor the [`--ecl-output-double-precision`](https://github.com/OPM/opm-common/blob/1ac240a425c64c3b045d28fd963141e04d333f90/src/opm/output/eclipse/RestartIO.cpp#L492) flag (default `false`), so the vectors will typically be output as single precision (`float`) values.

In the extension of this work I'm also looking to simplify the flag management to make it easier for users to request the exact semantics they want.  As of right now, the double precision flag is unconditionally treated as `false` when creating ECLIPSE-compatible output files.  I'm planning to lift this restriction, thereby allowing those who do not need the ability to restart ECLIPSE from Flow to output variables as double precision if they so chose.  We might even consider turning off unit conversion for that case, but that proposition is a little more tenuous, especially for restarting Flow from an existing Flow base run.